sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeWithPrebuiltModel.md

```diff
@@ -0,0 +1,260 @@
+# Analyze a document with a prebuilt model
```

Comment:
We had a "Strongly typing a recognized form" sample in V2. I think it may be worth bringing it back in beta 2. Thoughts?

Reply:
There are plans to strongly type the prebuilts given that the service now returns the field schemas.
I think we should wait until we design that piece and see how close we get to the "strongly type recognizer form sample"